pandas/_libs/tslibs/base.pxd

```diff
@@ -0,0 +1,20 @@
+from cpython.datetime cimport datetime, timedelta
+
+cdef class ABCTimedelta(timedelta):
```

Comment:
ic, so now we have a cdef class that we can do instance on w/o having to import the world, nice.